keyboards/ymdk/ymd60/info.json

```
@@ -0,0 +1,232 @@
+{
+    "keyboard_name": "Oni HHKB",
+    "manufacturer": "YMDK",
+    "maintainer": "Pangorin",
+    "usb": {
+        "vid": "0x45D4",
+        "pid": "0x0021",
+        "device_version": "0.0.1"
```

migrated from `config.h`
```suggestion
        "device_version": "0.0.1",
        "force_nkro": true
```